src/core_modules/capture-core/components/TrackedEntityTypeSelector/TrackedEntityTypeSelector.component.js

```diff
@@ -59,17 +59,23 @@ const InfoOutlinedIconWithStyles = withStyles({
     },
 })(InfoOutlinedIcon);
 
-export const Index =
-  ({ trackedEntityTypesWithCorrelatedPrograms, classes, onSelect, selectedSearchScopeId }: Props) =>
-      (<>
+export const TrackedEntityTypeSelectorPlain =
+  ({ classes, onSelect, selectedSearchScopeId }: Props) => {
+      const trackedEntityTypesWithCorrelatedPrograms = useTrackedEntityTypesWithCorrelatedPrograms();
+
+      const handleSelectionChange = ({ selected }) => {
+          onSelect(selected, scopeTypes.TRACKED_ENTITY_TYPE);
+      };
```

Comment:
Moved this up here coz later on we would like to dispatch some actions from here